ci-operator/config/redhat-operator-ecosystem/playground/redhat-operator-ecosystem-playground-master.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  base:
+    name: ubi
+    namespace: ocp
+    tag: "7"
+
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+
+images:
+- dockerfile_path: Dockerfile
+  from: bin
+  inputs:
+    bin:
+      as:
+        - builder
+    base:
+      as:
+        - runtime
+  to: redhat-operator-ecosystem-playground
+
+binary_build_commands: make build
+
+promotion:
+  namespace: ci
+  tag: latest
+
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+
+tests:
+- as: unit
+  commands: make test-unit
+  container:
+    from: src
+- as: perf
+  commands: make test-perf
+  container:
+    from: src
+- as: e2e
+  commands: make test-e2e
```

Comment:
This test won't work without `tag_specification` (that's the reason why the rehearsal fails for it). `tag_specification` specifies the "rest of openshift", essentially the version of openshift to install.

I think for now it is fine to put something usual like:

```
tag_specification:
  namespace: ocp
  name: "4.4"
```

Which will make the jobs install current 4.4 OpenShift latest greatest (images continuously built from the HEAD of `master` branches). This will allow you to iterate on the `make test-e2e` target in the playground repo, while we'll be inventing ways how to make the openshift versions parametrized for you.

Reply:
Thanks -- added.